src/papilio/vendors/keyboardio/model01.cpp

```diff
@@ -0,0 +1,44 @@
+/* -*- mode: c++ -*-
```

Comment:
This seems to be pretty much the same as `src/kaleidoscope_simulator/vendors/keyboardio/model01.cpp`, sans the namespace.